examples/with-jest-babel/pages/_app.tsx

```diff
@@ -1,4 +1,4 @@
-import '../styles/global.css'
+import 'styles/global.css'
```

Comment:
Why does styles not have an @ though and the others do? Let's be consistent with one way 👍

Reply:
No problem, this is addressed now as well!

I forget to convert all of these because I find the `@/` path pattern a bit unnecessary in general. (`"baseUrl": "."` in itself is pretty clean IMO), but for consistency's sake, I didn't mind.

I guess the argument is that you can easily see what's your module and what's from `node_modules` and I don't argue with that. :+1: